internal/build/imgsrc/ensure_builder.go

```diff
@@ -325,92 +401,113 @@ func createBuilder(ctx context.Context, org *fly.Organization, region, builderNa
 		return nil, nil, fmt.Errorf("waiting for app %s: %w", app.Name, retErr)
 	}
 
-	var volume *fly.Volume
-	numRetries := 0
-	for {
-		volume, retErr = flapsClient.CreateVolume(ctx, fly.CreateVolumeRequest{
-			Name:                "machine_data",
-			SizeGb:              fly.IntPointer(50),
-			AutoBackupEnabled:   fly.BoolPointer(false),
-			ComputeRequirements: &guest,
-			Region:              region,
-		})
-		if retErr == nil {
-			region = volume.Region
-			break
+	config := &fly.MachineConfig{
+		Env: map[string]string{
+			"ALLOW_ORG_SLUG": org.Slug,
+			"LOG_LEVEL":      "debug",
+		},
+		Guest: &guest,
+		Image: p.image(),
+	}
+
+	if buildkit {
+		config.Services = []fly.MachineService{
+			{
+				InternalPort: 1234,
+				Ports:        []fly.MachinePort{{Port: fly.IntPointer(1234)}},
+			},
```

Comment:
we may want to set autostart `on` and autostop to `stop` in this config- at least that's the config I've been using that seems to be convenient.